grails-app/migrations/join_dets_and_rxrs.groovy

```diff
@@ -272,4 +272,57 @@ databaseChangeLog = {
                    viewName: 'invalid_detection'
         )
     }
+
+    changeSet(author: "jburgess", id: "1430268900000-09") {
+        createView("""
+            SELECT
+              station_station_name AS station,
+              installation_name AS installation,
+              rxr_project_name AS project,
+              aatams.scramblepoint(st_makepoint(station_longitude, station_latitude)) AS public_location,
+              st_x(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lon,
+              st_y(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lat,
+              ('http://localhost:8080/aatams/installationStation/show/'::text || station_id) AS installation_station_url,
```

Comment:
TBH, I wondered the same thing.  Take a look at https://github.com/aodn/aatams/blob/38e5f8cec160d2790673796cd33fe65a3b5f3d3b/grails-app/migrations/rebased_db_initialisation.groovy#L2134 though, and you will see that this is (hopefully) equivalent to what was there before.  I didn't want to change it without investigating...  Might be worth raising an issue on this one though, seems wrong to me.